$NetBSD: patch-aa,v 1.2 2015/12/29 23:34:57 dholland Exp $

Don't use threads. (XXX: why not?)
Use -c with install as install without -c doesn't behave sensibly.

--- MKDefs.in.orig	Sun May  7 20:20:11 2000
+++ MKDefs.in
@@ -12,10 +12,10 @@
 VSLVER=@VSLISP_MAJOR_VERSION@.@VSLISP_MINOR_VERSION@-@VSLISP_MICRO_VERSION@
 
 CC= @CC@ 
-OPT = @CFLAGS@ @XX_CFLAGS@ -DUSE_THREADS -c
+OPT = @CFLAGS@ @XX_CFLAGS@ -c
 OSOPT = -DOS='"@target_os@"' -DCPU='"@target_cpu@"'
 LOPT= -o
-INSTALL= install
+INSTALL= install -c
 
 PREFIX= @prefix@
 VSLIBPATH= @prefix@/lib
